test/functional/feature_anchors.py

```diff
@@ -78,6 +81,60 @@ def run_test(self):
         self.log.info("When node starts, check if anchors.dat doesn't exist anymore")
         assert not os.path.exists(node_anchors_path)
 
+        self.log.info("Ensure addrv2 support")
+        # Use proxies to catch outbound connections to networks with 256-bit addresses
+        onion_conf = Socks5Configuration()
+        onion_conf.auth = True
+        onion_conf.unauth = True
+        onion_conf.addr = ('127.0.0.1', 10000)
+        onion_conf.keep_alive = True
+        onion_proxy = Socks5Server(onion_conf)
+        onion_proxy.start()
+        self.restart_node(0, extra_args=[f"-onion={onion_conf.addr[0]}:{onion_conf.addr[1]}"])
+
+        self.log.info(f"Add 256-bit-address block-relay-only connections to node")
+        ONION_ADDR = "pg6mmjiyjmcrsslvykfwnntlaru7p5svn6y2ymmju6nubxndf4pscryd.onion:8333"
+        self.nodes[0].addconnection(ONION_ADDR, 'block-relay-only')
+
+        self.log.info("Check peer info")
+        peer_info = self.nodes[0].getpeerinfo()
+        assert len(peer_info) == 1
```

Comment:
I think you could use `assert_equal` here and in the other similar cases

```suggestion
        assert_equal(len(peer_info), 1)
```

Reply:
sure, done thanks!